.travis.yml

```diff
@@ -47,8 +47,7 @@ install:
   - hash -r;
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
-  - echo $TRAVIS_OS_NAME
-  - echo $CONDA_PYTHON
+  - echo "Python $CONDA_PYTHON running on $TRAVIS_OS_NAME"
```

Comment:
@cclauss 
* IMHO, this would look more compact.
* If `CONDA_PYTHON=nightly`, it would be OK? What do you think?

Reply:
Seems OK to me.

Reply:
* Alright then. Let me merge this too.
* Thanks for your opinion. :smiley: